starknet_py/constants.py

```diff
@@ -9,3 +9,6 @@
 
 ROOT_DIR = os.path.realpath(os.path.join(os.path.dirname(__file__), "."))
 DEFAULT_CPP_LIB_PATH = os.path.join(ROOT_DIR, "utils", "crypto")
+
+MAINNET_ETH_CONTRACT = "0xc662c410C0ECf747543f5bA90660f6ABeBD9C8c4"
```

Comment:
Maybe it would be good to document where these addresses come from? As a comment or in docs.